src/linked-project-columns.js

```diff
@@ -3,17 +3,12 @@ const octokit = require('@octokit/graphql');
 const queries = require('./graphql');
 const utils = require('./utils');
 
-const AUTOMATION_NOTE_HEADER = `
-**DO NOT EDIT**
-This column is automatically populated from the following columns:
-`.trim();
-const AUTOMATION_NOTE_COLUMN_REFERENCE = `
-- [<project name>'s '<column name>' column](<column url>).
-`.trim();
+const AUTOMATION_NOTE_HEADER = `**DO NOT EDIT**: Synced with [this column](<column url>)`
```

Comment:
The syncing automation handles syncing multiple source columns into a single target column.  This change would unfortunately no longer list out multiple source columns.

There is already an option to turn off the automated header and the ability to manually add a card that is ignored by this automation, which can be combined to replace the default verbose header with a custom one.

I think it would also be helpful to add an option to the automation that would put individualized header cards in the target column for each source column like
```
- header for source 1
- source 1 item
- source 1 item
- header for source 2
- source 2 item
- header for source 3
- source 3 item
```

The individual header cards could also be turned off or replaced with cards that better matched the user's workflow.